doc/10.-Proxy-settings.md

```diff
@@ -0,0 +1,46 @@
+# Proxy settings
+
+The proxy manager allows AD administrators to enforce proxy settings on the clients. Currently, only system-wide proxy settings are supported.
+
+Proxy settings are configurable under the following GPO path:
+
+* System-wide level, located in `Computer Configuration > Policies > Administrative Templates > Ubuntu > Client management > System proxy configuration`
```

Comment:
An image here showing the navigation bar on the GPO configuration window would be nice.

Maybe we can use the current image for it and then add another one with the actual configuration window opened and some example content?